src/makeInfinityAdder.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 
 /***
- *
  * @return {function}
  */
-
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let result = 0;
+
+  const adder = (a) => {
+    if (typeof a !== 'number') {
```

Comment:
The check for 'typeof a !== 'number'' is not sufficient to determine if no parameters were passed. You should specifically check if 'a' is 'undefined' to handle the case where the function is called without parameters.